### PR TITLE
Add zIRC to the libraries list

### PR DESCRIPTION
**Pull request by wolfy1339**

### CODE REVIEW

File: _data/sw_libraries.yml [hunk 1]
```diff
@@ -138,3 +138,16 @@
...
+      # ref: https://github.com/itslukej/zirc/blob/master/zirc/caps.py
+      #      https://github.com/itslukej/zirc/blob/master/zirc/ext/sasl.py
+      #      https://github.com/itslukej/zirc/issues/9
+      link: https://github.com/itslukej/zirc
+      support:
+        v3.1:
+          - cap
+          - sasl
+        v3.2:
+        - message-tags
```
Comment (DanielOaks) [inline _data/sw_libraries.yml]: This line should also be indented like the 3.1 lines are

Comment (wolfy1339) [inline _data/sw_libraries.yml]: I put it there because another library had it listed at the same spot

Comment (attilamolnar) [inline _data/sw_libraries.yml]: Right, that's an oversight. Will be fixed shortly.

**Comment by DanielOaks** [comment]
Thanks very much!

**Pull request merged**